developer_key_publishing.md

```diff
@@ -0,0 +1,176 @@
+    2. `keytocard` and select `1` to set the signature key on the YubiKey.
+    7. `keytocard` and select `3` to set the authentication key on the YubiKey.
+    8. `quit` and save your changes.
+
+See the [official YubiKey documentation](https://support.yubico.com/support/solutions/articles/15000006420-using-your-yubikey-with-openpgp)
+
+#### Verify your signing key
+
+Use the following command as a quick way to verify that your new key can be used to produce a valid signature:
+
+```bash
+gpg -u ${USER_NAME}@clabs.co -o - --sign <(head -c 256 /dev/urandom) | gpg --verify -
+```
+
+## Key Publishing
+
+Keys are published to [celo.org](https://celo.org) using [OpenPGP WKD](https://gnupg.org/blog/20161027-hosting-a-web-key-directory.html), which is essentially just a hosted folder of public keys. Published keys are managed with Github by submitting a pull request to the `celo-monorepo`.
+
+#### Configuring your key
+
+The domain (e.g. `celo.org`) of the user identifier much match the site it's hosted on, so you'll need to run the following command to add the `celo.org` domain to your key:
+
+```bash
+gpg --quick-add-uid ${USER_NAME}@clabs.co "${REAL_NAME} <${USER_NAME}@celo.org>"
+```
+
+Additionally you may want to add identities for `staging.celo.org` and `dev.celo.org` so you can confirm your keys are published correctly on a non-production version of the website.
+
+```bash
+gpg --quick-add-uid ${USER_NAME}@clabs.co "${REAL_NAME} <${USER_NAME}@staging.celo.org>"
+gpg --quick-add-uid ${USER_NAME}@clabs.co "${REAL_NAME} <${USER_NAME}@dev.celo.org>"
+```
+
+> Note: You do not need an `@celo.org` email address to publish a key with the `@celo.org` domain, but it must not conflict with an existing email or published key.
+
+#### Adding your key to the repository
+
+Adding your key to the `@celo/web` package of `celo-monorepo` will allow them to be available on [celo.org](https://celo.org) with the next website deployment.
+
+Running the following command from the root of `celo-monorepo` will add your key to the `packages/web/openpgpkey` directory:
+
+```bash
+gpg --list-options show-only-fpr-mbox -k ${USER_NAME}@clabs.co | $(gpgconf --list-dirs libexecdir)/gpg-wks-client -v --install-key -C packages/web/openpgpkey
+```
+
+You should confirm that new key files were added to the `packages/web/openpgpkey/` directory and open a pull request with the changes. You should see new files in the `packages/web/openpgpkey/${DOMAIN}/hu` folders for each of the domains you added to you key, including `clabs.co`, `celo.org`, `staging.celo.org` and `dev.celo.org` if you followed the directions above.
+
+#### Verifying the published key
+
+Once a new version of [celo.org](https://celo.org) is published, you will be able to verify your keys are published correctly with the key lookup instructions below.
+
+Verifying your keys are correctly added to the repository can be accomplished by pushing your changes to `dev.celo.org`.
```

Comment:
only people at cLabs can do this right now and only if they have access to gcloud.

Reply:
The primary audience for this guide is cLabs employees and contractors because we don't have any other core developers for now, so I want to keep this section, but I amended it to be more explicit that your need access to the Google Cloud project.